app/views/assessor_interface/application_forms/index.html.erb

```diff
@@ -30,7 +30,7 @@
     </section>
 
     <section id="app-applications-filters-reference">
-      <%= f.govuk_text_field :reference, label: { size: "s" }, placeholder: I18n.t("helpers.placeholder.assessor_interface_filter_form.reference") %>
+      <%= f.govuk_text_field :reference, label: { size: "s" } %>
```

Comment:
is this code missing the text for the label?

<%= f.govuk_text_field :reference, label: { size: "s", text: I18n.t("helpers.assessor_interface_filter_form.reference") } %>

Reply:
No, strings are automatically taken from locale files if they exist:

https://govuk-form-builder.netlify.app/building-blocks/localisation/